startup.sh

```diff
@@ -8,9 +8,14 @@ export WINEDEBUG=-all
 
 winetricks dxvk || true
 
-if [ ! -f "NorthstarLauncher.exe" ]; then
-    curl -s https://api.github.com/repos/R2Northstar/Northstar/releases/latest | grep browser_download_url | cut -d '"' -f 4 | xargs wget -O northstar.zip
-    unzip northstar.zip
-fi                                                           
-                                                             
+API_DATA=$(curl -s https://api.github.com/repos/R2Northstar/Northstar/releases/latest)
+LATEST_VER=$(echo "$API_DATA" | grep tag_name | cut -d '"' -f 4)
+VERFILE="northstar-version.txt"
+
+if [ -f "$VERFILE" ] && [ "$LATEST_VER" != cat "$VERFILE" ] ||  then
```

Comment:
Something like the following could be more appropriate

```suggestion
if  ( [ -f "$VERFILE" ] && [ "$LATEST_VER" != $(cat $VERFILE) ] ) || [ ! -f "NorthstarLauncher.exe" ]
then